rest/data/project.go

```diff
@@ -50,6 +53,38 @@ func FindProjectById(id string, includeRepo bool, includeProjectConfig bool) (*m
 	return p, nil
 }
 
+// RequestAWSAccess creates a JIRA ticket that requests build to create an AWS user for the specified project.
+func RequestAWSAccess(projectIdentifier string) error {
+	if projectIdentifier == "" {
+		return errors.New("project identifier cannot be empty")
+	}
+	settings, err := evergreen.GetConfig()
+	summary := fmt.Sprintf("Create AWS key for s3 uploads for %s project", projectIdentifier)
+	description := fmt.Sprintf("Could you create an s3 key for the new [%s|%s/project/%s/settings/general] project?", projectIdentifier, settings.Ui.UIv2Url, projectIdentifier)
+	jiraIssue := message.JiraIssue{
```

Comment:
are these tickets usually created with default visibility, or as private?

Reply:
We might want to double check that and maybe open a conversation about it if that's the case.

Reply:
Default visibility should be fine; there's nothing private in this ticket subject matter.